### DISCUSSION TITLE
New placeholder (%block-data%)

### DISCUSSION THREAD

**Discussion by Reeferx7** [PlaceholderAPI/PlaceholderAPI#916] [(3) Ideas]
Hello, is it possible to add a new placeholder to check the placed block data (when a player looking/interact with block)?

Like anvil face direction, and other blocks screen example how the placeholder must return:

![εικόνα](https://user-images.githubusercontent.com/8740881/218890270-136a5471-dfdd-4c0a-8a48-3d3efafdf7d4.png)

**Comment by Andre601** [comment]
This isn't the place for placeholder requests, as said in your closed issue.

**Comment by Reeferx7** [comment]
sorry for that!!!! it says ideas , where can i request for placeholders ? i cant find it.

edit: delete my post i found the place. Really sorry for waste your time.
